test/file.ts

```diff
@@ -2715,6 +2715,61 @@ describe('File', () => {
         });
       });
 
+      it('should process the entire stream', done => {
+        tmp.setGracefulCleanup();
+        tmp.file(async (err, tmpFilePath) => {
+          assert.ifError(err);
+
+          const fileContents = 'abcdefghijklmnopqrstuvwxyz';
+
+          fileReadStream.on('resume', () => {
+            fileReadStream.emit('data', fileContents);
```

Comment:
Minor: for future reviewers, we may want to leave a comment as to why we're explicitly emitting twice

Reply:
Do you think the test title is sufficient? If not, we should change the test title to something more descriptive.

Reply:
I think the title is sufficient, describes what it is testing succinctly enough.